implementations/rust/ockam/ockam_command/src/identity/show.rs

```diff
@@ -18,9 +21,28 @@ impl ShowCommand {
     pub fn run(self, options: CommandGlobalOpts) -> anyhow::Result<()> {
         let cfg = options.config;
         let node = get_final_element(&self.node_opts.api_node);
-        let port = cfg.get_node_port(node);
-
-        connect_to(port, self, show_identity);
+        if let Some(port) = cfg.try_get_node_port(node) {
```

Comment:
@mrinalwadhwa I think we should not create a `Vault` here for this. Couldn't we just use the `Rpc` command structure here?
1. Send the request to the background node if `node` exists
2. Else, create an embedded node and send the request to it

Reply:
For my edification, why not create a Vault here?